src/components/footer.js

```diff
@@ -55,12 +59,20 @@ const Footer = () => {
         </div>
         <div className="dropdown-menu">
           <div className="dropdown-content">
-            <Link to="/" className="dropdown-item">
-              English
-            </Link>
-            <Link to="/zh" className="dropdown-item">
-              简体中文
-            </Link>
+            {Object.keys(langConfig.languages).map((lang) => (
+              <Link
+                key={lang}
+                className="dropdown-item"
+                to={window.location.pathname.replace(
```

Comment:
"window" is not available during server side rendering. Make sure you pass the build process before committing.

Reply:
You can use `useLocation` to get the location of the current path.